components/eam/src/physics/rrtmg/radiation.F90

```diff
@@ -362,13 +372,13 @@ end function radiation_nextsw_cday
 
 !================================================================================================
 
-  subroutine radiation_init(phys_state)
```

Comment:
Will these changes conflict with an upstream merge of E3SM?

Reply:
Sorry I let this go. I think it only creates a conflict if E3SM also modified these lines. This is a very SCREAM-specific thing though, I don't think any of the SPA infrastructure is in E3SM upstream, so I'm not sure if this change should get pushed back upstream. Thoughts?

Reply:
We do have scream-specific things in other parts, like Homme. One way to isolate them is to do something like
```
#ifdef SCREAM
  ! scream impl
#else
  ! eam impl
#endif
```